docs/Using Fleet/CIS-Benchmarks.md

```diff
@@ -90,9 +90,13 @@ fleetctl apply --policies-team "Workstations" -f cis-policy-queries.yml
 ```
 
 ## Limitations
-Although Fleet is able to cover a large majority of benchmarks, Fleet's current set of benchmarks only implements benchmark auditing steps that can be automated.
 
-For a list of specific checks which are not covered by Fleet, please visit the README for each benchmark.
+Certain benchmarks require human action to audit, and cannot be automated by Fleet. For a list of specific checks which are not covered, please visit the README for each benchmark:
```

Comment:
```suggestion
Certain benchmarks require human action to audit, and cannot be automated by a policy in Fleet. For a list of specific benchmarks which are not covered, please visit the README for each benchmark:
```

Suggest throwing the word "policy" in here to familiarize users.

Just spoke w/ Sharon and he's aligned w/ simplifying "manual" => "not covered by Fleet"